nginx/ssl/README.txt

```diff
@@ -0,0 +1,14 @@
+Hello Sysadmin,
```

Comment:
This documentation should exist in the wiki under deployment. Additionally, any explicit setup outside of docker should be scripted and documented in the wiki. Venue is a great example of this.